gubernator/filters.py

```diff
@@ -62,6 +63,21 @@ def rep(m):
                                 flags=re.MULTILINE))
 
 
+def do_testcmd(name):
+    if name.startswith('k8s.io/'):
+        if name.count(' ') != 1:
```

Comment:
Done.

On Wed, Jun 8, 2016 at 6:34 PM, Joe Finney <notifications@github.com> wrote:

> In gubernator/filters.py
> <https://github.com/kubernetes/test-infra/pull/128#discussion_r66370282>:
>
> > @@ -62,6 +63,21 @@ def rep(m):
> >                                  flags=re.MULTILINE))
> >
> >
> > +def do_testcmd(name):
> > +    if name.startswith('k8s.io/'):
> > +        if name.count(' ') != 1:
>
> OK, mind adding a comment saying as much?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/test-infra/pull/128/files/eea1154ddbd157d20e4ac81a1cfde3d3ea10b0d5#r66370282>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAM7nO0x9Kpu0jVgvt-czz0JC7HBodE4ks5qJ22WgaJpZM4Ixjp2>
> .
>